src/compiler/checker.ts

```diff
@@ -258,6 +259,10 @@ namespace ts {
             return emitResolver;
         }
 
+        function getEmitResolverNoTypeCheck() {

```

Comment:
Im wondering do we need this, maybe we can just pass undefined as a resolver and do a check in `emitFile` that resolver is defined (since this is the only codepath that will use it)